ee-9/feature-pack/pom.xml

```diff
@@ -440,7 +440,31 @@
             <scope>provided</scope>
         </dependency>
 
-        <!-- module and copy artifact dependencies -->
+	<!-- module and copy artifact dependencies -->
+
+           <dependency>
+                <groupId>org.antlr</groupId>
+                <artifactId>antlr4</artifactId>
+		<scope>provided</scope>
+                <exclusions>
+                    <exclusion>
+                        <groupId>*</groupId>
+                        <artifactId>*</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+            
+            <dependency>
+                <groupId>org.antlr</groupId>
+                <artifactId>antlr4-runtime</artifactId>
+		<scope>provided</scope>
+                <exclusions>
+                    <exclusion>
+                        <groupId>*</groupId>
+                        <artifactId>*</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
```

Comment:
Can you move these 2 down to the "EE 9 Impls" section?

Sorry, I'm kind of nitpicky about this file. :)

Reply:
moved.